LanguageFeatures/Inline-classes/static_analysis_member_invocation_A06_t02.dart

```diff
@@ -2,29 +2,27 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion We say that an inline class declaration DV has a member named n
-/// in the case where DV declares a member named n, and in the case where DV has
-/// no such declaration, but DV has a direct superinterface V that has a member
-/// named n. In both cases, the member declaration named n that DV has is said
-/// declaration.
+/// @assertion We say that an extension type declaration DV has a non-extension
+/// type member named n in the case where DV does not declare a member named n,
+/// but DV has a direct extension type superinterface V that has a non-extension
+/// type member named n, or DV has a direct non-extension type superinterface T
+/// whose interface contains a member signature named n.
 ///
-/// @description Checks that if inline class `V` has a superinterface with a
-/// member `m` then this member is also presents in `V`
+/// @description Checks that if an extension type `ET` has a superinterface with
+/// a member `m` then this member is also presents in `ET`
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=inline-class
 
 import "../../Utils/expect.dart";
 
-inline class V1 {
-  final int id = 0;
-  V1();
+class I {
+  int id = 0;
 }
 
-inline class V implements V1 {
-  V();
-}
+extension type ET(int id) implements I {}
```

Comment:
Same issue: `implements I` is an error, we'd need something like `implements Comparable<num>` (just to try something new ;-), and then we should be able to do `e.compareTo(1.5)` where `e` has static type `ET`.

Reply:
Alternatively, we could check things like these ones:

```dart
class I {
  int i = 0;
}

class J extends I {
  int j = 1;
}

extension type E(J rep) implements I {
  int get jOfE => rep.j;
}

void main() {
  var e = E(J());
  e.rep; // OK.
  e.i; // OK.
  e.j; // Compile-time error.
  e.jOfE; // OK.
}
```